test/Microsoft.ML.Tests/TrainerEstimators/LbfgsTests.cs

```diff
@@ -53,8 +53,8 @@ public void TestLogisticRegressionStats()
             var stats = linearModel.Statistics;
             LinearModelStatistics.TryGetBiasStatistics(stats, 2, out float stdError, out float zScore, out float pValue);
 
-            CompareNumbersWithTolerance(stdError, 0.250672936);
-            CompareNumbersWithTolerance(zScore, 7.97852373);
+            Assert.Equal(0f, stdError);
```

Comment:
Why did you change the baseline here?

Reply:
This test was making use of the Math.Net numeric implementation. Now that it got removed, the stdError is not being calculated anymore. 

Let me see if i need to change any code comments to make that more obvious. 

---
In reply to: [236375885](https://github.com/dotnet/machinelearning/pull/1722#discussion_r236375885) [](ancestors = 236375885)